janus_core/calculations/md.py

```diff
@@ -607,7 +617,12 @@ def _run_dynamics(self) -> None:
             if self.logger:
                 self.logger.info("Beginning temperature ramp at %sK", temps[0])
             for temp in temps:
+                if temp == 0.0:
+                    continue
                 self.temp = temp
+                self._set_velocity_distribution()
+                if self.ensemble != "nve":
```

Comment:
I don't think it should make a difference, but might it be safer to check against the class, as we do elsewhere (e.g. NVT_NH)?

Reply:
yap good point I will update